### PR TITLE
setup-java v3

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by sullis** [comment]
cc: @pderop

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,10 +7,11 @@ jobs:
...
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
     - name: Set up JDK
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3
       with:
         java-version: 13
+        distribution: temurin
```
Comment (pderop) [inline .github/workflows/ci.yml:15]: @sullis ,

the `temurin` distribution does not seem to support java13, which is required for tests, 

see https://github.com/reactor/BlockHound/actions/runs/5391516225/jobs/9788493900?pr=361

could you revise your PR with another distribution ? or maybe remove the temurin distrib for jdk13 ?

thank you.

File: .github/workflows/snapshot.yml [hunk 2]
```diff
@@ -9,11 +9,12 @@ jobs:
...
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: 13 # Modern JVM is needed for Java 10+ specific tests
+          distribution: temurin
```
Comment (pderop) [inline .github/workflows/snapshot.yml:17]: same issue here, temurin does not support jdk13.

**Comment by sullis** [comment]
Outdated.

**Pull request closed without merging**